docs/Build/Meson_build.md

```diff
@@ -1,36 +1,64 @@
 # Build and Installation Instructions with Meson
 
+Meson is a modern and fast build system with a lot of features. You can
+find its documentation at [mesonbuild.com](https://mesonbuild.com/).
+
 ## Install Dependencies
 
-Fedora:
-```bash
-dnf install meson qt5-qtbase-devel rtaudio-devel "pkgconfig(jack) help2man"
-```
+=== "Fedora"
 
-Debian/Ubuntu:
-```bash
-apt install meson build-essential qtbase5-dev librtaudio-dev libjack-jackd2-dev help2man
-```
+    ```bash
+    dnf install meson qt5-qtbase-devel rtaudio-devel "pkgconfig(jack)" help2man
+    ```
 
-MacOS with brew (not tested):
-```bash
-brew install meson qt rt-audio jack help2man
-```
+=== "Debian/Ubuntu"
+
+    ```bash
+    apt install meson build-essential qtbase5-dev librtaudio-dev libjack-jackd2-dev help2man
+    ```
+
+=== "MacOS"
+
+    ```bash
+    brew install meson qt rt-audio jack help2man
+    ```
+
+    Meson might not find qt when installed with brew. But brew tells you to set
+    PKG_CONFIG_PATH to a directory where it finds qt's pkgconfig file.
+    This directory has to be set as additional pkgconfig path in meson:
+
+    ```bash
+    meson --buildtype release --pkg-config-path PATH_GIVEN_BY_BREW build_release
```

Comment:
I think the pkg-config path can be obtained with `` `brew --prefix qt5`/lib/pkgconfig ``

Reply:
That's neat. Should I make one command out of it?

Reply:
Do you want to test the changed command? I don't have MacOS.

Reply:
Tested on macOS with the updated instructions - everything works as expected